README.md

```diff
@@ -16,7 +16,8 @@ print("Ultimate Python study guide")
 [한국어](README.ko.md) |
 [繁体中文](README.zh_tw.md) |
 [Español](README.es.md) |
-[Deutsch](README.de.md)
+[Deutsch](README.de.md) |
+[हिन्दी](README.hi.md)
```

Comment:
Put this in the following README files:
- README.ko.md
- README.zh_tw.md
- README.es.md
- README.de.md
If there are any others I missed, please put them in there as well.

Reply:
Sure, Ill update all the readme files with the reference.